clients/drcachesim/tests/scheduler_unit_tests.cpp

```diff
@@ -160,6 +166,101 @@ verify_scheduler_stats(scheduler_t::stream_t *stream, int64_t switch_input_to_in
            migrations);
 }
 
+// Returns a string with one char per input.
```

Comment:
It returns a vector of strings, so can you describe the vector aspect of the return value?

Reply:
Changed to:

// Returns a vector of strings, one per ouput, where each string has one char per input
// showing the order of inputs scheduled onto that output.